README.md

```diff
@@ -37,3 +40,42 @@ document.body.appendChild(new BlueButton);
 ```
 
 This polyfill does that automatically because all builtin extends must be query-able via `querySelectorAll`, but other browsers won't do that automatically.
+
+
+## Examples for how to do feature detection for this polyfill
+
+Your first option would be, to extend and register an element, appending it do a DocumentFragment and then checking if it's both an instance of a Custom Element as well as a Builtin Element.
+
+```js
+// This returns false, if Customized Builtin-Elements are not supported
+function supportsCustomizedBuiltInElements() {
+  try {
+    class CustomButton extends HTMLButtonElement {}
+    customElements.define('test-button', CustomButton, { extends: 'button' });
+
+    const fragment = new DocumentFragment();
```

Comment:
you don't need any of this in here ... `new CustomButton` is enough to fail.

Reply:
You mean, without defining it as a customElement, like a class instance as an extension to HTMLButtonElement, or just the Fragment part? I'm pretty sure I need to register it to instantiate it.

Reply:
once registered, you don't need fragments around to fail ... also, `test-button` is too easy to register ... I'd go with `'feature-detection-' + crypto.randomUUID()` instead, and call it a day.

Reply:
that being said, I think all of this should be instead in the original polyfill, not here ... it's from there you complained about lack of Safari only documentation, onc you are here you probably already know all your options ... what do you think?